ledger/src/blockstore/blockstore_purge.rs

```diff
@@ -866,22 +865,22 @@ pub mod tests {
             .put(1, &index1)
             .unwrap();
 
-        let statuses: Vec<_> = blockstore
+        let num_statuses = blockstore
             .transaction_status_cf
-            .iterator_cf_raw_key(IteratorMode::Start)
-            .collect();
-        assert_eq!(statuses.len(), 15);
+            .iter(IteratorMode::Start)
```

Comment:
I started off wanting to kill `iterator_cf_raw_key()` altogether. However, we still need it for the cases where we perform compaction.

Regardless, I opted to use `iter()` instead of `iterator_cf_raw_key()` to work towards being able to remove the test only `iterator_cf_raw_key()` function